velox/docs/functions/presto/array.rst

```diff
@@ -49,6 +49,10 @@ Array Functions
         SELECT array_distinct(ARRAY [1, 2, 1]); -- [1, 2]
         SELECT array_distinct(ARRAY [1, NULL, NULL]); -- [1, NULL]
 
+.. function:: array_dupes(array(E)) -> boolean
```

Comment:
@amitkdutta  Adding these functions to Presto was a mistake. These names are not in line with the rest of the functions. I suggest we remove these from Velox and add mapping to Prestissimo if needed.

CC: @kevinwilfong